gradle/dependencies.gradle

```diff
@@ -13,7 +13,7 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '144.0.0'
+      mapboxNavigatorVersion = '145.0.0'
```

Comment:
I think we can also add a changelog line about updated EV data being used in continuous alternatives requests.